libs/positions/src/lib/positions-table.tsx

```diff
@@ -327,7 +329,7 @@ export const PositionsTable = forwardRef<AgGridReact, Props>(
             node,
           }: PositionsTableValueFormatterParams & {
```

Comment:
chore (non blocking): I also added some type helpers to ui-toolkit/components/ag-grid which make defining types here a bit cleaner